### ISSUE TITLE
Adaptive Probe Volumes

### ISSUE THREAD

**Issue by MiMoer** [Dustyroom/flat-kit-doc#119]
It doesn't work when Adaptive Probe Volumes are enabled.
![image](https://github.com/user-attachments/assets/8ca5f1c1-bf95-42b3-af81-5368217c6106)
![image](https://github.com/user-attachments/assets/2332b106-be24-4072-a447-22bc6182ed94)

Flat Kit version 4.7.5
Unity 6000.0.23f1
Dev platform: WindowsEditor
Target platform: StandaloneWindows64
URP installed: True, version 17.0.3
Render pipeline: UniversalPipeline
Render pipeline asset: [FlatKit] Example URP Asset
Color space: Linear
Quality config: [FlatKit] Example URP Asset
Graphics config: [FlatKit] Example URP Asset

**Comment by vdzitsiuk** [comment]
Hi,

Is the issue reproducible in a new scene of a fresh project? If so, what would be the steps to do it?

**Comment by MiMoer** [comment]
> Hi,
> 
> Is the issue reproducible in a new scene of a fresh project? If so, what would be the steps to do it? Please add as many details as you can.

Yes, I reproduced the error in a newly created project, and it fails simply by toggling this setting.

![image](https://github.com/user-attachments/assets/3a971991-2155-47f2-8707-938ec58be68b)
![image](https://github.com/user-attachments/assets/9e8678fb-abf8-492f-9ab2-eafd41485217)
![image](https://github.com/user-attachments/assets/cf45e9e0-ace2-4ce7-bc2a-2b12acc14243)

`
Shader error in 'FlatKit/Stylized Surface': 'SampleProbeVolumePixel': no matching 0 parameter function at Assets/FlatKit/Shaders/StylizedSurface/LibraryUrp/LitForwardPass_DR.hlsl(122) (on d3d11)

Compiling Subshader: 0, Pass: ForwardLit, Fragment program with DR_RIM_ON DR_SPECULAR_ON PROBE_VOLUMES_L1 _ADDITIONAL_LIGHTS _ADDITIONAL_LIGHT_SHADOWS _DETAILMAPBLENDINGMODE_MULTIPLY _GRADIENTSPACE_WORLD _MAIN_LIGHT_SHADOWS_CASCADE _SHADOWS_SOFT _TEXTUREBLENDINGMODE_MULTIPLY _UNITYSHADOWMODE_MULTIPLY
Platform defines: SHADER_API_DESKTOP UNITY_ENABLE_DETAIL_NORMALMAP UNITY_ENABLE_REFLECTION_BUFFERS UNITY_LIGHTMAP_FULL_HDR UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_PBS_USE_BRDF1 UNITY_PLATFORM_SUPPORTS_DEPTH_FETCH UNITY_SPECCUBE_BLENDING UNITY_SPECCUBE_BOX_PROJECTION UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS
Disabled keywords: DEBUG_DISPLAY DIRLIGHTMAP_COMBINED DR_CEL_EXTRA_ON DR_ENABLE_LIGHTMAP_DIR DR_GRADIENT_ON DR_VERTEX_COLORS_ON DYNAMICLIGHTMAP_ON FOG_EXP FOG_EXP2 FOG_LINEAR INSTANCING_ON LIGHTMAP_ON LIGHTMAP_SHADOW_MIXING LOD_FADE_CROSSFADE PROBE_VOLUMES_L2 SHADER_API_GLES30 SHADOWS_SHADOWMASK UNITY_ASTC_NORMALMAP_ENCODING UNITY_COLORSPACE_GAMMA UNITY_FRAMEBUFFER_FETCH_AVAILABLE UNITY_HARDWARE_TIER1 UNITY_HARDWARE_TIER2 UNITY_HARDWARE_TIER3 UNITY_LIGHTMAP_DLDR_ENCODING UNITY_LIGHTMAP_RGBM_ENCODING UNITY_METAL_SHADOWS_USE_POINT_FILTERING UNITY_NO_DXT5nm UNITY_NO_SCREENSPACE_SHADOWS UNITY_PBS_USE_BRDF2 UNITY_PBS_USE_BRDF3 UNITY_PRETRANSFORM_TO_DISPLAY_ORIENTATION UNITY_UNIFIED_SHADER_PRECISION_MODEL UNITY_VIRTUAL_TEXTURING _ADDITIONAL_LIGHTS_VERTEX _ALPHAPREMULTIPLY_ON _ALPHATEST_ON _BASEMAP_PREMULTIPLY _CELPRIMARYMODE_CURVE _CELPRIMARYMODE_SINGLE _CELPRIMARYMODE_STEPS _CLUSTERED_RENDERING _DBUFFER_MRT1 _DBUFFER_MRT2 _DBUFFER_MRT3 _DETAILMAPBLENDINGMODE_ADD _DETAILMAPBLENDINGMODE_INTERPOLATE _EMISSION _FORWARD_PLUS _GRADIENTSPACE_LOCAL _LIGHT_COOKIES _LIGHT_LAYERS _MAIN_LIGHT_SHADOWS _MAIN_LIGHT_SHADOWS_SCREEN _NORMALMAP _RECEIVE_SHADOWS_OFF _SCREEN_SPACE_OCCLUSION _TEXTUREBLENDINGMODE_ADD _UNITYSHADOWMODE_COLOR _UNITYSHADOW_OCCLUSION _WRITE_RENDERING_LAYERS
`

`
Shader error in 'FlatKit/Stylized Surface': syntax error: unexpected integer constant at Assets/FlatKit/Shaders/StylizedSurface/LibraryUrp/LitForwardPass_DR.hlsl(122) (on d3d11)

Compiling Subshader: 0, Pass: ForwardLit, Fragment program with DR_RIM_ON DR_SPECULAR_ON PROBE_VOLUMES_L1 _ADDITIONAL_LIGHTS _ADDITIONAL_LIGHT_SHADOWS _DETAILMAPBLENDINGMODE_MULTIPLY _GRADIENTSPACE_WORLD _MAIN_LIGHT_SHADOWS_CASCADE _SHADOWS_SOFT _TEXTUREBLENDINGMODE_MULTIPLY _UNITYSHADOWMODE_MULTIPLY
Platform defines: SHADER_API_DESKTOP UNITY_ENABLE_DETAIL_NORMALMAP UNITY_ENABLE_REFLECTION_BUFFERS UNITY_LIGHTMAP_FULL_HDR UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_PBS_USE_BRDF1 UNITY_PLATFORM_SUPPORTS_DEPTH_FETCH UNITY_SPECCUBE_BLENDING UNITY_SPECCUBE_BOX_PROJECTION UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS
Disabled keywords: DEBUG_DISPLAY DIRLIGHTMAP_COMBINED DR_CEL_EXTRA_ON DR_ENABLE_LIGHTMAP_DIR DR_GRADIENT_ON DR_VERTEX_COLORS_ON DYNAMICLIGHTMAP_ON FOG_EXP FOG_EXP2 FOG_LINEAR INSTANCING_ON LIGHTMAP_ON LIGHTMAP_SHADOW_MIXING LOD_FADE_CROSSFADE PROBE_VOLUMES_L2 SHADER_API_GLES30 SHADOWS_SHADOWMASK UNITY_ASTC_NORMALMAP_ENCODING UNITY_COLORSPACE_GAMMA UNITY_FRAMEBUFFER_FETCH_AVAILABLE UNITY_HARDWARE_TIER1 UNITY_HARDWARE_TIER2 UNITY_HARDWARE_TIER3 UNITY_LIGHTMAP_DLDR_ENCODING UNITY_LIGHTMAP_RGBM_ENCODING UNITY_METAL_SHADOWS_USE_POINT_FILTERING UNITY_NO_DXT5nm UNITY_NO_SCREENSPACE_SHADOWS UNITY_PBS_USE_BRDF2 UNITY_PBS_USE_BRDF3 UNITY_PRETRANSFORM_TO_DISPLAY_ORIENTATION UNITY_UNIFIED_SHADER_PRECISION_MODEL UNITY_VIRTUAL_TEXTURING _ADDITIONAL_LIGHTS_VERTEX _ALPHAPREMULTIPLY_ON _ALPHATEST_ON _BASEMAP_PREMULTIPLY _CELPRIMARYMODE_CURVE _CELPRIMARYMODE_SINGLE _CELPRIMARYMODE_STEPS _CLUSTERED_RENDERING _DBUFFER_MRT1 _DBUFFER_MRT2 _DBUFFER_MRT3 _DETAILMAPBLENDINGMODE_ADD _DETAILMAPBLENDINGMODE_INTERPOLATE _EMISSION _FORWARD_PLUS _GRADIENTSPACE_LOCAL _LIGHT_COOKIES _LIGHT_LAYERS _MAIN_LIGHT_SHADOWS _MAIN_LIGHT_SHADOWS_SCREEN _NORMALMAP _RECEIVE_SHADOWS_OFF _SCREEN_SPACE_OCCLUSION _TEXTUREBLENDINGMODE_ADD _UNITYSHADOWMODE_COLOR _UNITYSHADOW_OCCLUSION _WRITE_RENDERING_LAYERS
`
